README.md

```
@@ -1,695 +1,160 @@
-```typescript
-// app/services/health.ts
-import { type GetApiHealthResponse, zGetApiHealthResponse } from '#shared/types/api';
-
-export const getHealthApi = async (): Promise<GetApiHealthResponse> => {
-  const response = await $fetch<GetApiHealthResponse>('/api/health', {
-    method: 'GET',
-  });
-  return zGetApiHealthResponse.parse(response); // Zodバリデーション
-};
-```
+3. **[ハイブリッド状態管理](./docs/state-management.md)** - TanStack Query + Pinia
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)